src/core/trackingmodel.cpp

```diff
@@ -117,7 +117,7 @@ bool TrackingModel::setData( const QModelIndex &index, const QVariant &value, in
   return true;
 }
 
-bool TrackingModel::featureInTracking( QgsVectorLayer *layer, const QgsFeatureId featureId )
+bool TrackingModel::featureInTracking( QgsVectorLayer *layer, const QgsFeatureId &featureId )
```

Comment:
```suggestion
bool TrackingModel::featureInTracking( QgsVectorLayer *layer, QgsFeatureId featureId )
```

QgsFeatureId is a long int and better copied by value.